eve/workers/centos7_terraform/requirements.sh

```diff
@@ -0,0 +1,11 @@
+TERRAFORM_VERSION=0.11.8
+
+yum install -y epel-release unzip
+yum install -y python2-pip python2-devel python36 python36-devel haveged
+pip2 install tox
+
+curl -O https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /usr/local/sbin/
+
+systemctl start haveged
```

Comment:
I get we need this for now, however, I'll chat with the team to get `virtio-rng` support on our VMs ;-)

Reply:
oh, I do this semi-automatically, may be virtio-rng is in place, haven't check